dashboard/lib/tasks/seed.rake

```diff
@@ -27,6 +27,12 @@ namespace :seed do
   include TimedTask
   verbose false
 
+  timed_task check_migrations: :environment do
+    if defined? ActiveRecord
```

Comment:
what case is this covering? I'd think it would be safe to assume that ActiveRecord is always loaded within the `:environment` context